content/collections/reference/metadata.md

```diff
@@ -19,127 +19,133 @@ The examples given throughout this reference can be seen in context in the [comp
     description="Unique identifier derived from name (acronyms, dash-separated)."
     example="demo"
     required=true
->}}
+/>}}
 
 {{< configOption
     name="name"
     type="string"
     description="Display name of the collection."
     example="Demo Collection"
     required=true
->}}
+/>}}
 
 {{< configOption
     name="tagline"
     type="string"
     description="Concise description of collection topic."
     example="Services used by Open Terms Archive"
     required=true
->}}
+/>}}
 
 {{< configOption
     name="languages"
     type="array of strings"
     description="List of [ISO 639-1 (two-letter)](https://en.wikipedia.org/wiki/ISO_639) language codes representing languages allowed in the collection."
     example="[en, fr, de]"
     required=true
->}}
+/>}}
 
 {{< configOption
     name="jurisdictions"
     type="array of strings"
     description="List of [ISO 3166-2 country codes](https://en.wikipedia.org/wiki/ISO_3166-2) representing jurisdictions covered by the collection."
     example="[EU]"
     required=true
->}}
+/>}}
 
 {{< configOption
     name="description"
     type="string"
     description="Detailed description of the collection"
-    example=`    The **Demo** collection tracks changes to the terms of use of services used by Open Terms Archive.
-    
-    This provides a reference collection for best practices and enables the Open Terms Archive Core Team to be a user of the software it produces.
-    `
     required=false
 >}}
 
+```yaml
+description: >
```

Comment:
Why do we add the key? Other examples give only the value as example.

Reply:
When dealing with complex multiline examples, I find it more readable to include the key. However, for single values, it doesn’t seem necessary.